packages/bsky/src/hydration/hydrator.ts

```diff
@@ -563,9 +567,36 @@ const listUrisFromProfileViewer = (item: ProfileViewerState | null) => {
   if (item?.blockingByList) {
     listUris.push(item.blockingByList)
   }
+  if (item?.blockedByList) {
+    listUris.push(item.blockedByList)
+  }
```

Comment:
It's relatively clear how `mutedByList` and `blockingByList` play into profile hydration (since they appear in the profile view), but this one is a bit less apparent—maybe we add a comment about it.
```suggestion
  // blocked-by list does not appear in views, but will be used to evaluate the existence of a block between users.
  if (item?.blockedByList) {
    listUris.push(item.blockedByList)
  }
```

Reply:
good call 👍